tap_productboard/schemas/objectives.json

```diff
@@ -14,89 +14,55 @@
       "type": ["integer", "null"]
     },
     "parent": {
-      "anyOf": [
-        {
-          "type": "null"
+      "type": ["null", "object"],
+      "properties": {
+        "id": {
+          "type": "string"
         },
-        {
+        "links": {
           "type": "object",
           "properties": {
-            "id": {
-              "type": "string"
-            },
-            "links": {
-              "type": "object",
-              "properties": {
-                "self": {
-                  "type": "string"
-                },
-                "html": {
-                  "type": "string"
-                }
-              }
-            }
-          },
-          "required": ["id"]
+            "self": { "type": "string" },
+            "html": { "type": "string" }
+          }
         }
-      ]
+      },
+      "required": ["id"]
     },
     "owner": {
       "type": "object",
       "properties": {
-        "email": {
-          "type": "string"
-        }
+        "email": { "type": "string" }
       }
     },
     "timeframe": {
-      "anyOf": [
-        {
-          "type": "null"
-        },
-        {
-          "type": "object",
-          "properties": {
-            "startDate": {
-              "anyOf": [
-                {
-                  "type": "string",
-                  "format": "date"
-                },
-                {
-                  "const": "none"
-                }
-              ]
-            },
-            "endDate": {
-              "anyOf": [
-                {
-                  "type": "string",
-                  "format": "date"
-                },
-                {
-                  "const": "none"
-                }
-              ]
+      "type": ["null", "object"],
+      "properties": {
+        "startDate": {
+          "anyOf": [
+            {
+              "type": "string",
+              "format": "date"
             },
-            "granularity": {
-              "type": ["string", "null"]
-            }
-          }
+            { "const": "none" }
+          ]
+        },
+        "endDate": {
+          "anyOf": [{ "type": "string", "format": "date" }, { "const": "none" }]
```

Comment:
@mianazmatali9 This `anyOf` syntax is used through these JSON files, hotglue will always just look at the first entry here. The better way to allow `null` is using syntax like this:
```
"type": ["string", "null"]
```

Reply:
```json
{
    "id": "b7743db1-7f2a-4ac5-ad79-e0abd9d88e24",
    "name": "Limit bookings per day",
    "archived": false,
    "timeframe": {
        "startDate": "none",
        "endDate": "none",
        "granularity": "none"
    },
    "createdAt": "2021-06-11T19:55:19.606168Z",
    "updatedAt": "2023-07-21T02:46:55.859048Z"
}
```

@hsyyid  We're receiving "none" as a string from Productboard. Please review the last commit, where I’ve converted "none" to null values and updated the schemas to support null.